\startcomponent chapter1

\product thesis \environment layout


\Chapter{Introduction}


\Section{Structural biology in the Omics-age}

Since the start of modern-day Western science, Man is on a mission to
thoroughly study Nature in order to understand, manipulate, and overcome her
\cite[Nietzsche1891]. Above all a fundamental insight into life is a hallmark
in the whole scientific enterprise, which is biologically represented in its
irreducable form by the cell. The cell is a highly complex system that is
regarded as the building block of life and is able to reproduce itself
indepently. Even though DNA holds a full blueprint of an organism, cells
themselves are mainly organized by proteins and their interactions
\cite[Braun2012]. Recent technological and methodological advances have enabled
the inquiry of the interaction networks that are formed by proteins, and showed
that the set of all interacting protein-complexes, the interactome, is several
orders of magnitude larger than the total number of proteins that the genome
encodes for, the proteome \cite[Stein2011].

The field of structural biology tries to understand the workings of the
molecules of life by studying their structure, preferable up to atomic
resolution, as this provides a functional and mechanical description of the
system \cite[Campbell2002]. The latter can be achieved by high-resolution
methods, mainly X-ray crystallography and NMR spectroscopy. Unfortunately, both
methods are hampered by several limitions. X-ray crystallography is mainly
limited by the production of high-quality crystals, an undertaking that becomes
more difficult with increasing structure size and flexiblity of the
macromolecules; for NMR spectroscopy it is mainly the size of proteins that is
limiting structure determination, as spectra become heavily congested for
larger complexes, making peak assignment unfeasable. Furthermore, neither
method is amenable for high-througput investigations, a necessary requirement
for the structural elucidation of the interactome.

In order to close the structure knowledge gap, computational methods have been
deviced to aid in this quest. Homology modeling is a successful approach to
predict the structure of a complex with high-sequence identity to another
already known structure, and heavily extends the structural knowledge of the
proteome. Macromolecular docking is the field that occupies itself with
predicting the structure of a complex starting from their individual components \cite[Moreira2010]. 



\Section{Integrative modeling}

\Subsection{Sources of information}

\Subsubsection{Cryo-electron microscopy}

\Subsubsection{Cross-links/mass spectrometry}

\Subsection{Software packages and platforms}


\Subsection{HADDOCK}


\Section{Explorative modeling}


\Section{Overview of thesis}


\stopcomponent
